### ISSUE TITLE
openxlsx::openXL() does not work on linux box

### ISSUE THREAD

**Issue by jzadra** [IntegralEnvision/integral#21]
```
> openxlsx::openXL(wb)
Only Libreoffice/OpenOffice found; I'll use it.
/usr/lib/libreoffice/program/javaldx: error while loading shared libraries: libreglo.so: cannot open shared object file: No such file or directory
Warning: failed to read path from javaldx
/usr/lib/libreoffice/program/soffice.bin: error while loading shared libraries: libreglo.so: cannot open shared object file: No such file or directory

```
#https://github.com/rstudio/rstudio/issues/8539

**Comment by ebenp** [comment]
I think this has to do with system2 not being on the same bash path. Maybe

**Comment by ebenp** [comment]
Closed with c9fe9e1b6f087e53e7fb2db5aacfd0c33dac1d08
